### DISCUSSION TITLE
[Feature Request] Remove folder from project key binding

### DISCUSSION THREAD

**Discussion by nerdyman** [zed-industries/zed#14195] [Feature Requests]
Apologies if I've missed this but wasn't able to see any related issues.

I'd like to be able to bind `shift+cmd+w` to perform the same "Remove folder from workspace" functionality available in the folder context menu within the dock.

I've bound `shift+cmd+a` to `workspace::AddFolderToProject` and it works nicely but there doesn't seem to be mapping to remove folders.

Here's my keymap.json

```jsonc
[
  {
    "context": "Editor",
    "bindings": {
      "shift-cmd-a": "workspace::AddFolderToProject"
    }
  },
  // Doesn't seem to exist?
  {
    "context": "Editor",
    "bindings": {
      "shift-cmd-w": "workspace::RemoveFolderFromProject"
    }
  }
]
```

For reference, in vscode it shows an overlay to select the project to remove

![image](https://github.com/zed-industries/zed/assets/4908432/f8a4a202-febe-466f-b3ca-20268b3852cc)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
